docs/source/config-file.mdx

```diff
@@ -220,3 +204,43 @@ transport:
       - mcp
       - profile
 ```
+
+## Example config file
+
+The following example file sets your endpoint to `localhost:4001`, configures transport over Streamable HTTP, provides a GraphOS key and graph reference,
+enables introspection, and provides two local MCP operations for the server to expose.
+
+```yaml config.yaml
+endpoint: http://localhost:4001/
+transport:
+  type: streamable_http
+graphos:
+  apollo_key: <YOUR_APOLLO_KEY>
+  apollo_graph_ref: <YOUR_APOLLO_GRAPH_REF>
+introspection:
+  introspect:
+    enabled: true
+operations:
+  source: local
+  paths:
+    - relative/path/to/your/operations/userDetails.graphql
+    - relative/path/to/your/operations/listing.graphql
+```
+
+## Override configuration options using environment variables
+
+You can override configuration options using environment variables. The environment variable name is the same as the option name, but with `APOLLO_MCP_` prefixed. You can use `__` to mark nested options.
+
+For example, to override the `introspection.execute.enabled` option, you can set the `APOLLO_MCP_INTROSPECTION__EXECUTE__ENABLED` environment variable.
+
+```sh
+APOLLO_MCP_INTROSPECTION__EXECUTE__ENABLED=true
```

Comment:
Did it work? 😮 I thought bools would throw an error.

```suggestion
APOLLO_MCP_INTROSPECTION__EXECUTE__ENABLED="true"
```

Reply:
Can confirm that double quotes are needed, thanks for the catch!